modules/core/src/utils/texture.js

```diff
@@ -28,9 +28,19 @@ export function createTexture(layer, image) {
     image = {data: image};
   }
 
+  const specialTextureParameters = {};
+  if (image.compressed) {
+    specialTextureParameters[gl.TEXTURE_MIN_FILTER] =
+      image.data.length > 1 ? gl.LINEAR_MIPMAP_NEAREST : gl.LINEAR;
+  }
+
   const texture = new Texture2D(gl, {
     ...image,
-    parameters: {...DEFAULT_TEXTURE_PARAMETERS, ...layer.props.textureParameters}
+    parameters: {
+      ...DEFAULT_TEXTURE_PARAMETERS,
+      ...layer.props.textureParameters,
+      ...specialTextureParameters
```

Comment:
Should this override any user-supplied parameters?

If so, it should be enforced in luma.gl's `Texture` constructor.

Reply:
At least in the non-compressed use case, the user should be allowed to set it to `NEAREST`.

Reply:
I changed order